docs/source/netutils/lib_mapping/index.rst

```diff
@@ -0,0 +1,115 @@
+*****************
+Library Mappings
+*****************
+
+These dictionaries provide mappings in expected vendor names between Netmiko, NAPALM, pyntc, ntc-templates, pyats, and scrapli.
```

Comment:
I was thinking more of a human side of the story, basically, describe the use cases where this may be interesting and show bi-directionally how this may work